assets/app/view/game/auto_action/auction_bid.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'view/game/auto_action/base'
+
+module View
+  module Game
+    module AutoAction
+      class AuctionBid < Base
+        needs :bid_target, store: true, default: nil
+
+        def name
+          "Auto bid in Auction Round#{' (Enabled)' if @settings}"
+        end
+
+        def description
+          'Automatically bid, buy or pass in the auction round. '\
+            'Bids increase by the minimum increment. '\
+            'Buys only when the current price matches your specified price.'
+        end
+
+        def render
+          return [] unless @game.round.auction?
+
+          form = {}
+
+          children = [h(:h3, name)]
```

Comment:
you can make this just a static array up to line 29 because it's all the same

childen = [...]